include/openflow/orange-ext.h

```diff
@@ -56,20 +56,33 @@ OFP_ASSERT(sizeof(struct ol_update_map) == 16);
 
 struct ol_dump_map_request {
     ovs_be16 filter_prog;  /* Filter program ID. */
-    ovs_be16 map_id; /* Map ID. */
+    ovs_be16 nb_maps;
+    /* Followed by:
+     *   - Exactly nb_maps map_ids of total nb_maps * 2(ovs_be16) bytes. */
```

Comment:
Why `2 * ovs_be16`?

Reply:
Because ovs_be16 has 2 bytes. I can change it if you want.
Exactly nb_maps * map_id size (2 bytes) bytes containing nb_maps map indentifiers. ?

Reply:
Oh. I see. Yes, I'd drop the `2` and the parentheses. Either `total nb_maps * 2 bytes` or `total nb_maps * ovs_be16`.

Reply:
Done.